### DISCUSSION TITLE
Is there a way to also get all nuget dependencies from project references and add it to the main project so that indirectly referenced nugets can also be resolved ?

### DISCUSSION THREAD

**Discussion by matlemuzic** [teroneko/Teronis.DotNet#25] [Ideas]
Is there a way to also get all nuget dependencies from project references and add it to the main project so that indirectly referenced nugets can also be resolved ?

Cheers.

**Comment by teneko** [comment]
I am not an oracel, but when you want to study a little bit of source code and have a kind of understanding of MSBuild, dotnet CLI, C# code and the the tools, frameworks and such it is using, you can take a look at this https://github.com/rabotcrypto/nethermind/blob/ecc6fa9de5b565f09d73f08873401ff2b557b0d4/rabot/tasks/ExternalProjectPackages.targets (used for example by https://github.com/rabotcrypto/nethermind/blob/ecc6fa9de5b565f09d73f08873401ff2b557b0d4/rabot/binding/RabotCrypto.Nethermind.Api/RabotCrypto.Nethermind.Api.csproj)

Simple said, it collects all external nuget packages, except those that are "normally" implictly derived from project references. I don't say this is the answer, but you can look around and try to figure out. Please do not waste to much time into it, it is really very complicated and confusing and you are better to just wait for an official solution. If you really have too much time you can download the repo at https://github.com/rabotcrypto/nethermind and play with the scripts you can find here: https://github.com/rabotcrypto/nethermind/tree/master/rabot/scripts, it may give you a feeling what all the stuff is actually doing...
